upbound_uxp/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+import os
+import subprocess
+from unittest import mock
+
+import pytest
+
+from datadog_checks.upbound_uxp import UpboundUxpCheck
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    print("conftest: dd_environment spinning up")
+    cmd = ('kind', 'get', 'clusters')
+    p = subprocess.run(cmd, capture_output=True, text=True)
```

Comment:
Try setting `timeout` to maybe 600 seconds then catch https://docs.python.org/3/library/subprocess.html#subprocess.TimeoutExpired and print the output

Reply:
I have added the timeout, but it did not run into the expiration. I believe that this setup step succeeds, and that github tests hang thereafter.

@pytest.fixture(scope='session')
def dd_environment():
    print("conftest: dd_environment spinning up")
    cmd = ('kind', 'get', 'clusters')
    try:
        p = subprocess.run(cmd, timeout=600, capture_output=True, text=True)
        cluster_present = False
        for cluster in p.stdout.split('\n'):
            if cluster == "uxp":
                print("conftest: dd_environment using existing uxp cluster")
                cluster_present = True
                break
        if not cluster_present:
            print("conftest: dd_environment: creating new uxp cluster")
            os.system("tests/fixtures/k8s-uxp-agent-check.sh")
        yield
        print("conftest: dd_environment cleaning up")
    except TimeoutExpired as ex:
        print(ex)